qa/kerberos/build.gradle

```diff
@@ -78,643 +82,366 @@ dependencies {
 // Testing jars
 // =============================================================================
 
-// Build uber cascading jar for testing
```

Comment:
Apologies in advance for the bad diff rendering - this is just the rest of the file wrapped in an if statement